main.cpp

```diff
@@ -42,6 +45,47 @@ std::vector<std::string> split(const std::string &s, char delim) {
     return elems;
 }
 
+void InitHomeMap(){
+    std::ifstream myfile("Mods\\CommandsMod\\home.txt");
+    if(!myfile.is_open()) {
+        return;
+    }
+    std::string line;
+
+    while (getline(myfile, line)) {
+        std::vector<std::string> elements = split(line, ' ');
+        if (elements.size() == 4) {
+            std::string name = elements.at(0);
+            std::string x = elements.at(1);
+            std::string y = elements.at(2);
+            std::string z = elements.at(3);
+            std::vector<long long> coords {std::stoll(x), std::stoll(y), std::stoll(z)};

```

Comment:
[Supposedly ](https://en.cppreference.com/w/cpp/string/basic_string/stol)stoll can throw an exception, so it should be handled.